device/mellanox/x86_64-mlnx_msn2740-r0/ACS-MSN2740/sai.profile

```diff
@@ -1,3 +1 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_2740.xml
```

Comment:
@noaOrMlnx is it by intention to inject `SAI_DUMP_MFT_CFG_PATH` to SKUs like this?

Reply:
Yes